pilot/pkg/model/gateway_test.go

```diff
@@ -109,9 +113,17 @@ func TestMergeGateways(t *testing.T) {
 			"http-http2-server-config",
 			[]config.Config{gwHTTPWildcard, gwHTTP2Wildcard},
 			1,
-			2,
+			1,
```

Comment:
This is not related to this change. This was never being validated because test case issue.